app/auth/auth_service.ts

```diff
@@ -5,6 +5,8 @@ import { grp } from "../../proto/group_ts_proto";
 import { context } from "../../proto/context_ts_proto";
 import capabilities from "../capabilities/capabilities";
 
+const SELECTED_GROUP_ID_LOCAL_STORAGE_KEY = "buildbuddy://auth/selected_group_id";
```

Comment:
i still can't get over these local storage keys 😂 feels so heavyweight that i need a spec to learn how to generate them

Reply:
Lol, my concerns about localStorage would probably be better addressed by having a localStorage constants list somewhere, so I'll just make this smaller so it's at least easier to type